src/Uno.UI.RemoteControl/HotReload/MetadataUpdater/ElementUpdaterAgent.cs

```diff
@@ -0,0 +1,284 @@
+﻿// The structure of the ElementUpdaterAgent has been kept similar to the HotReloadAgent, 
+// which is based on the implementation in https://github.com/dotnet/aspnetcore/blob/26e3dfc7f3f3a91ba445ec0f8b1598d12542fb9f/src/Components/WebAssembly/WebAssembly/src/HotReload/HotReloadAgent.cs
+
+
+
+#if NET6_0_OR_GREATER || __WASM__ || __SKIA__
```

Comment:
I think this is always true?

Reply:
I'll remove these from this agent and the hotreloadagent in next PR